templates/geosafe/analysis/dynamic_scripts/create/main.html

```diff
@@ -369,7 +369,9 @@
         $.get('{% url 'geosafe:layer-tiles' %}', {'layer_id': layer_id}, function (data) {
 
             //default use of the marker
-            var marker = L.marker([data['layer_bbox_y0'], data['layer_bbox_x0']], {
+            center_y = data['layer_bbox_y0'] + ((data['layer_bbox_y1'] - data['layer_bbox_y0'] ) / 2);
+            center_x = data['layer_bbox_x0'] + ((data['layer_bbox_x1'] - data['layer_bbox_x0'] ) / 2);
+            var marker = L.marker([center_y, center_x], {
```

Comment:
Intead counting it ourself, what if we use `LatLngBounds` method and then get the center from it? So it would be something like this

```suggestion
            corner_1 = L.latLng(data['layer_bbox_y0'], data['layer_bbox_x0'])
            corner_2 = L.latLng(data['layer_bbox_y1'], data['layer_bbox_x1'])
            var marker = L.marker(L.latLngBounds(corner_1, corner_2).getCenter(), {
                  title:data['layer_name']
            });
```

cc @lucernae

Reply:
it works too @myarjunar 
i'll take your suggestion then.
thanks

Reply:
cool :+1: